### DISCUSSION TITLE
does the bitcore support dogecoind?

### DISCUSSION THREAD

**Discussion by panchengj** [dogecoin/dogecoin#3179] [Q&A]
does the bitcore support dogecoind?

https://github.com/bitpay/bitcore

**Comment by patricklodder** [comment]
Not out of the box. A couple of years ago I have patched some of it to at least be able to serialize headers and blocks received over p2p protocol, but I've never published this because there was no one to review. I also didn't maintain it through versions.

If you're looking for a working / maintained node.js interface, have a look at https://github.com/ShibeTechnology/dogecoin-spv-node, maybe it will suit your needs. If it doesn't and you want to work on bitcore, let me know where you're working on it and I'll see if any of the code that I already did for myself is still applicable and help you.
